cmd/create/service/cmd.go

```diff
@@ -236,6 +249,23 @@ func run(cmd *cobra.Command, argv []string) {
 		})
 	}
 
+	// Ensure all flags were used.
+	var unusedFlags bool
+	cmd.Flags().VisitAll(func(flag *pflag.Flag) {
+		reporter.Infof("%q", flag.Name)
```

Comment:
Not sure we want to keep this in here?

Reply:
You're right, I was using that for debugging